website/content/docs/concepts/billing-start-date.mdx

```diff
@@ -0,0 +1,14 @@
+---
+layout: docs
+page_title: Billing start date
+description: >-
+   A description of Vault cluster's billing start date used throughout Vault.
+---
+
+# Billing start date
+
+This is the billing start date associated with the reporting cluster. This will be the license start date if not configured.
```

Comment:
It would be useful if we actually defined what the billing start date is as the opening sentence. Saying that billing start date is the "billing start date associated with the reporting cluster" doesn't really tell me anything useful as a reader.